app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -1260,18 +1264,24 @@ public static synchronized BgReading bgReadingInsertLibre2(double calculated_val
                 bgReading.sensor_uuid = sensor.uuid;
                 bgReading.raw_data = raw_data;
                 bgReading.age_adjusted_raw_value = raw_data;
-                bgReading.filtered_data = raw_data;
```

Comment:
What happens to the `raw_data` parameter here?

Reply:
I'm using the `filtered_data` field to store the filtered (smoothed), but uncalibrated Libre2 values. I need those historical uncalibrated values for future noise estimation, and as `filtered_data` is only used by the Dexcom code path as far as I can tell, that field seemed like a good place to store the data.